internal/models/reservation_model.go

```diff
@@ -74,10 +92,10 @@ type AWSReservation struct {
 	AWSReservationID string `db:"aws_reservation_id" json:"aws_reservation_id"`
 
 	// The ID of the image from which the instance is created. AMI's must be prefixed with 'ami-'.
-	ImageID string `json:"image_id"`
+	ImageID string `db:"image_id" json:"image_id"`
 
 	// Detail information is stored as JSON in DB
-	Detail *AWSDetail `db:"detail" json:"detail"`
+	Detail *AWSDetail `json:"detail"`
```

Comment:
Why did you remove the 'db'?

Reply:
Oh this was not intentional, fixing. I was playing around with this, it did not work well with pgx/sqlx and I left a TODO comment on two functions which I had to implement. If we used pgx/scanny natively it would work out of box as pgx driver supports all of the postgres native types (this is JSONB).